functions/FunctionBase.py

```
@@ -310,36 +282,24 @@ def drawCostPaths(self, rows, con, args, geomType, canvasItemList, mapCanvas):
                     for pt in geom.asPolyline():
                         rubberBand.addPoint(pt)
 
-        # TODO label the edge instead of labeling the target points
+                # Label the edge
+                midPoint = PgrQ.getMidPoint()
+                midPointstr = midPoint.as_string(con)
```

Ruff (0.14.0)

Local variable `midPointstr` is assigned to but never used

Remove assignment to unused variable `midPointstr`

(F841)